tools/test/audio/std_utils/level_dbfs.m

```diff
@@ -6,15 +6,14 @@
 % x - signal
 %
 % Output
-% dbfs - sigmal level in dBFS
+% dbfs - signal level in dBFS
 %
 
 % SPDX-License-Identifier: BSD-3-Clause
 % Copyright(c) 2017 Intel Corporation. All rights reserved.
 % Author: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>
 
 %% Reference AES 17 3.12.3
-level_ms = mean(x.^2);
-dbfs = 10*log10(level_ms + 1e-20) + 20*log10(sqrt(2));
+dbfs = 20*log10(rms(x) * sqrt(2));
```

Comment:
Does rms() add the similar small value to avoid log of zero?

Reply:
Yes, `rms()`  additionally adds a small value for log zero. Because rms are equal to or higher than the average 'avg'.

I investigated the nonlinear nature of the log function and discovered that` log()` expands the smallest value while compressing the largest value.